distribution/server-x/src/main/content/bin/kc.sh

```diff
@@ -41,7 +42,11 @@ do
           fi
           ;;
       --config-file)
-          SERVER_OPTS="$SERVER_OPTS -Dkeycloak.config.file=$2"
+          SERVER_OPTS="$SERVER_OPTS -Dkc.config.file=$2"
```

Comment:
This is something we need to discuss because there we also say that ENV vars should be prefixed with `KC_`, so to keep consistency across both ENV and PROP I decided to use the `kc` prefix and drop `keycloak`.

It is a good point and I'm good with both approaches, although `kc` is shorter.